offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/persons/CustomerDbConnector.groovy

```diff
@@ -55,11 +56,6 @@ class CustomerDbConnector implements CreateCustomerDataSource, UpdateCustomerDat
   CustomerDbConnector(ConnectionProvider connectionProvider) {
     this.connectionProvider = connectionProvider
   }
-
-  @Override
-  void updateCustomerAffiliations(String customerId, List<Affiliation> affiliations) {
-    throw new RuntimeException("Method not implemented.")
-  }
   
   @Override
   List<Customer> findCustomer(String firstName, String lastName) throws DatabaseQueryException {
```

Comment:
This should be deprecated now, does it?

Reply:
I mean, with searching for active customers we actually only want that, right?

Reply:
this was just removed because i merged two branches containing the same function (once implemented, once not)

Reply:
I meant the lower part function `findCustomer`. We want to use the new function `findActiveCustomer` now and not the old one, right? Therefore I would put `findCustomer` to deprecated in the interface

Reply:
I see, good idea